src/fn/csv-to-docs.js

```diff
@@ -210,6 +210,10 @@ function parseBool(b) {
   else return b.toLowerCase() === 'true';
 }
 
+function calcRelTimestamp(b) {
```

Comment:
single letter variables (although still common in medic-conf) are not particularly desirable